core/src/main/java/org/apache/oozie/command/wf/WorkflowXCommand.java

```diff
@@ -85,6 +85,7 @@ protected void generateEvent(WorkflowActionBean wfAction, String wfUser) {
                     wfAction.getStatus(), wfUser, wfAction.getName(), wfAction.getStartTime(), wfAction.getEndTime());
             event.setErrorCode(wfAction.getErrorCode());
             event.setErrorMessage(wfAction.getErrorMessage());
+            event.setHadoopId(wfAction.getExternalId());
```

Comment:
Actually externalId is not hadoop id. Right?

Reply:
I had checked in our Oozie database & they seemed to match.
Do you think that is incorrect? How can I fetch `HadoopId` then?